lib/Url/Profile.php

```diff
@@ -601,8 +604,8 @@ protected function buildQuery()
                 $query = $relation->completeQuery(
                     $query,
                     $this->getColumnNameWithAlias('relation_'.$relation->getIndex()),
-                    $this->getArticleClangId(),
-                    $this->getColumnNameWithAlias('clang_id')
+                    $this->getColumnNameWithAlias('clang_id'),
+					null
```

Comment:
Könntest du das hier einmal so anpassen und testen?

```php
                $query = $relation->completeQuery(
                    $query,
                    $this->getColumnNameWithAlias('relation_'.$relation->getIndex()),
                    $this->getColumnNameWithAlias('clang_id'),
                    $this->getArticleClangId()
                );
```

Reply:
Das hatte ich auch schon probiert. Wenn im Profil im Feld "In welchem Artikel erfolgt die Ausgabe" in der Sprache
a) "alle Sprachen" ausgewählt ist funktioniert dein Vorschlag.
b) nur eine Sprache ausgewählt ist funktioniert er nicht.

Details zu b):
- Mit deinem Vorschlag werden folgende Datensätze in der Datenbank gespeichert:
![Bildschirmfoto von 2019-05-24 21-12-42](https://user-images.githubusercontent.com/4131072/58351504-3f4a2380-7e69-11e9-8cfb-ad41ab4f36ec.png)
- Wenn wie von mir vorgeschlagen der Wert `null` übergeben wird sieht das Ergebnis so aus wie es sein soll (im Beispiel wurde die Sprache mit der ID 2 "Deutsch" ausgewählt):
![Bildschirmfoto von 2019-05-24 21-13-19](https://user-images.githubusercontent.com/4131072/58351554-5d178880-7e69-11e9-99ab-6f2f94cc10a1.png)